### ISSUE TITLE
Support for left and right quotation marks.

### ISSUE THREAD

**Issue by jeggao** [BelugaHaechi/US-International-Scientific-Keyboard#21]
**Is your feature request related to a problem? Please describe.**
In conveying science in plain text, the simple ASCII quotation marks (`"` and `'`) are often used. I wish, in accordance with the philosophy of this project, that left and right quotation marks `“`, `”`, `‘`, and `’` (U+201C, U+201D, U+2018, and U+2019) can be fit into the keyboard to accurately convey when a quote starts.

This is especially desirable in dumb, online text editors where you care about the perfection of the text (cough CA cough).

**Describe the solution you'd like**
I understand the keyboard is already quite full but I hope the keys can be placed around `<` and `>`. This is due to many linguistic reasons such as the french `«` Guillemets  `»` as well as compose keys using them to achieve these characters.

An alternative solution might be to put them near brackets, which are used currently to type Guillemets. 

**Describe alternatives you've considered**
The alternative is to type everything in MS Word, and then paste it in the plain text editor. This is tedious, as it requires switching back and forth between a browser and Word, which is not allowed during special circumstances such as tests.

**Additional context**
None.

**Comment by BelugaHaechi** [comment]
Thanks for your request! In fact, the typographic quotation marks you have mentioned is a prescribed new feature in upcoming version 1.7.0, which I have resolved to placing in the dead keys of the AltGr states the original quotation marks. I also included other typographic quotation marks (such as the inverted ones in German). The secondary mapping would be left and right brackets `[]{}`, since in the currently layout they are used to type guillemets and also will be consistent with other paired punctuation. 

The new version may be released in a few days :)
